docs/reference/commandline/swarm_init.md

```diff
@@ -50,6 +55,14 @@ $ docker swarm init --listen-addr 192.168.99.121:2377 --auto-accept worker
 Swarm initialized: current node (1m8cdsylxbf3lk8qriqt07hx1) is now a manager.
 ```
 
+### `--cert-expiry`
```

Comment:
Pretty sure these description are either not useful at all or wrong :angel: 
@aaronlehmann @stevvooe I need your input :bow: